src/uct/ib/base/ib_iface.c

```diff
@@ -24,6 +24,7 @@
 #include <ucs/debug/log.h>
 #include <ucs/time/time.h>
 #include <ucs/sys/sock.h>
+#include <infiniband/mad.h>
```

Comment:
why needed? seems it can fail the build in some envs

Reply:
leftover, tried to use value from this header before